index.html

```diff
@@ -16,22 +18,21 @@
             <nav class="navbar-desktop">
                 <div class="social-media-icons">
                     <ul class="social-media-icons-ul">
-                        <li class="individual-social-media-icons"></li>
-                        <li class="individual-social-media-icons"></li>
+                        <li class="individual-social-media-icons"><i class="fa-brands fa-facebook-f"></i></li>
+                        <li class="individual-social-media-icons"><i class="fa-brands fa-twitter"></i></li>
                         <li class="individual-social-media-icons"><a href="#">English</a></li>
                         <li class="individual-social-media-icons"><a href="#">My Page</a></li>
                         <li class="individual-social-media-icons"><a href="#">Logout</a></li>
                     </ul>
                 </div>
                 <div class="desktop-brand-and-nav-links-div">
-                    <div ><a href="" class="logo">Pottery</a></div>
+                    <div ><a href="about-page.html" class="logo-anchor"><img src="images/logo-bg.png" class="logo" alt="safari logo"></a></div>
                     <ul class="nav-links-ul">
-                        <li class="nav-links"><a href="#">About</a></li>
+                        <li class="nav-links"><a href="about-page.html">About</a></li>
                         <li class="nav-links"><a href="#">Retreat</a></li>
                         <li class="nav-links"><a href="#">Join</a></li>
-                        <li class="nav-links"><a href="#">Sponsors</a></li>
-                        <li class="nav-links"><a href="#">News</a></li>
-                        <li class="nav-links"><a href="#">CC Campaign</a></li>
+                        <li class="nav-links"><a href="#">Gallery</a></li>
+                        <li class="nav-links"><a href="#">Accomodation</a></li>
```

Comment:
- [ ] I think it can be a good idea if you add the link to the home page to your mobile menu. When navigating between the two pages there is no way to return to the home page in the menu when we are on the about page. Please kindly fix that 🙃 🙃 

<img width="451" alt="Screen Shot 2023-01-12 at 14 02 51" src="https://user-images.githubusercontent.com/74501523/212050937-0e98dfe8-b7b9-452b-a800-473ba8135658.png">